test/models/calendar/calendar_title_type_test.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class CalendarTitleTypeTest < ActiveSupport::TestCase
+  test 'title_id should be unique based on type_id' do
+    title_type = calendar_title_types(:one)
+    dup_title_type = title_type.dup
+    title_type.status = 1
+    assert_not title_type.valid? && dup_title_type.valid?, 'Title be unique based on type'
```

Comment:
testlerde daime her bir assert veya refute tek bir şeyi kontrol etmeli. burada iki şeyi kontrol ediyorsun, bunları iki ayrı ifadeye ayırman gerekir. bu arada 'assert_not' (veya refute) unless gibi davranıyor - ve unless'in && ile nasıl çalıştığına bakalım:

```
puts "foo" unless true && false
> foo

puts "foo" unless false && false
> foo
```

hmmm.